README.md

```diff
@@ -1,4 +1,23 @@
 # wecount.inclusivedesign.ca
-The source files for the WeCount site
+The source files for the We Count website, built with [NuxtJS](https://nuxtjs.org/).
 
+## How to Run
+To run the website in local development mode, enter the following in the command line:
+```
+npm run dev
+```
+## How to Build
+To build a static version of the website, enter the following in the command line: 
+```
+nuxt generate
+```
+
+The static website's files will be in the newly-created `dist` directory in the root of the project.
+
+## How to Deploy
+This repository is connected to [Netlify](https://netlify.com). Updating the master branch will trigger a new build and deployment of the project to this temporary [url](https://wecount.netlify.com/).
```

Comment:
You can only do that with a paid version of netlify

Reply:
@TeddTech, that's not accurate; you can add a custom domain to the free version of Netlify. However, we can't do that yet because the WordPress site is using wecount.inclusivedesign.ca. So, we'd need to:

1. Change the WordPress site URL to something like https://cms.wecount.inclusivedesign.ca.
2. Change the API configuration in this site to point to the new WordPress URL.
3. Configure the DNS so that Netlify can serve this site at wecount.inclusivedesign.ca.

Also, the Netlify deploy is currently deploying your fork, @TeddTech, not the master branch. That needs to be changed before the site is made live.